### PR TITLE
fix: description error

### PR DESCRIPTION
**Pull request by jincheng9**
as titled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -708,7 +708,7 @@ hellomsg := `
 
 ## Reflection
 ### Type Switch
-A type switch is like a regular switch statement, but the cases in a type switch specify types (not values), and those values are compared against the type of the value held by the given interface value.
+A type switch is like a regular switch statement, but the cases in a type switch specify types (not values), and those types are compared against the type of the value held by the given interface value.
```
Comment (codingsince1985) [inline README.md:711]: nice. how about
```
A type switch is like a regular switch statement, but the cases in a type switch specify types (not values) which are compared against the type of the value held by the given interface value.
```

Comment (jincheng9) [reply README.md]: @codingsince1985 thanks,  I have modified this PR.

**Review by codingsince1985 — APPROVED** [review]
lgtm

**Comment by jincheng9** [comment]
@amiorin @basti1302 @mignev could you help reivew this PR. Thanks

**Comment by mignev** [comment]
lgtm r+

**Review by mignev — APPROVED** [review]
r+

**Review by codingsince1985 — APPROVED** [review]

**Review by a8m — APPROVED** [review]
💯

**Comment by jincheng9** [comment]
Thanks to all.

**Pull request merged**